javascript/input-and-string/js-inputs-and-strings_input-to-uppercase/js/index.js

```diff
@@ -10,3 +10,12 @@ Convert the input value to uppercase when clicking the button
 Hint: Access the value by using `.value` on the input element
 
 */
+const firstInput = document.querySelector('[data-js="first-input"]');
+const button = document.querySelector('[data-js="button-uppercase"]');
+const output = document.getElementById("help-text");
+
+button.addEventListener("click", (e) => {
+  e.preventDefault();
+  let text = firstInput.value.toUpperCase();
+  output.textContent = `${text}`;
```

Comment:
Du ersetzt hier diesen Text "Type something to test your code." mit dem Wert aus dem 1. Eingabefeld in Druckbuchstaben. Das war aber nicht die Aufgabe, sondern der Wert im 1. Eingabefeld soll in Druckbuchstaben wiedergegeben werden:

firstInput.value = text;